test/xpu_api/numerics/c.math/cmath.pass.cpp

```diff
@@ -101,7 +101,8 @@ void test_abs()
 #pragma clang diagnostic pop
 #endif
 
-    assert(dpl::abs(-1.) == 1);
+    assert(dpl::abs(-1.f) == 1.f);
+    IF_DOUBLE_SUPPORT(assert(dpl::abs(-1.) == 1));
```

Comment:
We check `-1.f` vs `1.f`, but then `1.` vs `1`.
I think either the first should compare to an integer or the second should compare to a double.  I don't quite understand which is the intended check here though.

Reply:
Fixed